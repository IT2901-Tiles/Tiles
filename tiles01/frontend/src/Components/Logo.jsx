import React from 'react';
import "../CSS/TopContainer.css";

function Logo() {
    return (
        <div className="Logo">
            <h1>TILES</h1>
        </div>
    );
}

export default Logo;