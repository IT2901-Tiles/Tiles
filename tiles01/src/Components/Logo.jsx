import React from 'react';
//import { Link } from "react-router-dom";


function Logo() {
    const styling ={
        backgroundColor : "#E53D2FEE",
        color: "#F3EAC2",
        borderRadius: "30px",
        fontFamily: 'Roboto',
        textShadow: "2px 2px #252525ab",
        border: "0.5px solid #72140C",
    }
    const textStyle ={
        padding: "0",
        position: "center",
        margin: "15px 0",
    }
    return (
        <div style={styling} className="Logo">
            <h1 style={textStyle}>TILES</h1>
        </div>

    );
}

export default Logo;