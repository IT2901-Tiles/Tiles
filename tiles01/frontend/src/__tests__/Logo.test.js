import {configure, shallow} from "enzyme"
import Adapter from 'enzyme-adapter-react-16';
import {cleanup} from '@testing-library/react';
import Logo from '../Components/Logo';

afterEach(() => {
    cleanup();
});

configure({
    adapter: new Adapter()
});

describe('Logo', () => {
    const wrapper = shallow(<Logo/>)
    it('contains a h1 element', () => {
        expect(wrapper.find('h1')).toHaveLength(1);
    });
    it('h1 renders correctly', () => {
        const heading = wrapper.find('h1');
        expect(heading.text()).toEqual('TILES');
    });
});