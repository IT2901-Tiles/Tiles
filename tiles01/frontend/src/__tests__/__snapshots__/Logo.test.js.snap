// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div
  className="Logo"
>
  <h1>
    TILES
  </h1>
</div>
`;
